src/main/msp/msp.c

```diff
@@ -3961,23 +3960,44 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
                     textVar = currentControlRateProfile->profileName;
                     break;
 
+                case MSP2TEXT_CUSTOM_MSG:
+                    {   //handle custom msg from msp
+                        char customRawMessage[MAX_NAME_LENGTH*3] = {0};
+                        const uint8_t textLength = MIN(MAX_NAME_LENGTH*3, sbufReadU8(src));
+                        memset(customRawMessage, 0, textLength);
+
+                        for (unsigned int i = 0; i < textLength; i++) {
+                            customRawMessage[i] = sbufReadU8(src);
+                        }
+                        
+                        unsigned int pointer_offset = 0;
+                        for(unsigned int msg_cnt = 0;msg_cnt < 3;msg_cnt++)
+                        {
```

Comment:
Wrong identation and opening curly brace needs to be on the same line
https://betaflight.com/docs/development/CodingStyle#curly-braces
```suggestion
                        for (unsigned int msg_cnt = 0; msg_cnt < 3; msg_cnt++) {
                        {
```

Reply:
Thank you. I have fixed.